Zadanie 9.3 Velociraptor czy Triceratops/js/scripts.js

```diff
@@ -0,0 +1,16 @@
+// scripts.js
+
+var text = 'Velociraptor is a genus of herbivorous ceratopsid dinosaur that first appeared during the late Maastrichtian stage of the late Cretaceous period.';
+var dinosaur = 'Triceratops is a genus of herbivorous ceratopsid dinosaur that first appeared during the late Maastrichtian stage of the late Cretaceous period.';
+var triceratopsUpperCased = dinosaur.replace('Triceratops','Triceratops'.toUpperCase());
```

Comment:
na wielkie litery powinienes zamienic to, co we wczesniejszej zmiennej, a nie stringa 'Triceratops'